### PR TITLE
tldjs typing [Types-2.0]

### PR DESCRIPTION
**Pull request by geoffreak**
case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

---

Typing for [`tldjs`](https://github.com/oncletom/tld.js)

### CODE REVIEW

File: tldjs/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     * It's usually a preliminary check before trying to use getDomain or anything else
+     * Beware: it does not check if the TLD exists.
+     * @param host The host/url to check
+     * @return {boolean}
+     */
+    isValid(host: string): string | null;
+}
+
+declare var tldjs: tldjs;
+export = tldjs;
```
Comment (mhegazy) [inline tldjs/index.d.ts]: Please note that your change is correct. this is more of a style change to simplify the file.

Comment (geoffreak) [inline tldjs/index.d.ts]: Done 👍

**Comment by mhegazy** [comment]
thanks. waiting on the CI server

**Comment by geoffreak** [comment]
Looks like `winjs` and `microsoft-live-connect` caused a CI failure

**Comment by mhegazy** [comment]
Fix queued in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11719

**Comment by geoffreak** [comment]
I've rebased this branch on your fix

**Pull request merged**